patches/chrome-installer-mac-signing-signing.py.patch

```diff
@@ -37,13 +48,12 @@ index b56853e5e2649328a81a2de20228de5abda08f11..f468b98a93ad85ed72be8ad82a4ee93c
      if part.sign_with_identifier:
          command.extend(['--identifier', part.identifier])
      reqs = part.requirements_string(config)
-@@ -233,6 +240,9 @@ def sign_chrome(paths, config):
- 
+@@ -234,6 +241,8 @@ def sign_chrome(paths, config):
      # Sign the framework bundle.
      sign_part(paths, config, parts['framework'])
-+    from signing_helper import GenerateWidevineSigFile
+ 
 +    GenerateWidevineSigFile(paths, config, parts['framework'])
```

Comment:
shouldn't this go above https://github.com/brave/brave-core/pull/2789/files#diff-bbb0120a0f4be9c6db810bfca1367681R53 ? Why are we calling it twice?

Reply:
@bridiver It's because adding sig file into framework bundle break framework bundle signing.
So, I had to sign framework bundle again after adding sig file.

Reply:
but that's my point, shouldn't we generate with widevine signature first before `sign_part`?

Reply:
also as mentioned before we shouldn't be calling two commands in a row in a patch. This method should be `sign_brave_part` or something and call `GenerateWidevineSigFile` and whatever else we need in the future inside that